### ISSUE TITLE
bluestone dust

### ISSUE THREAD

**Issue by DragonOfSapphire** [AlonzoVR/alonzo-bluestone-texture-pack#1]
The redstone dust is still red and not blue.

**Comment by AlonVDev** [comment]
Yes I know, I have tried ways of changing it but after taking a look at Minecraft's code, I realized the red tint is applied in the code, which also explains why the texture is white. However I will continue to try different ways of changing it.

**Comment by DragonOfSapphire** [comment]
Okay, I thought it was something simple that you missed. Sorry for bothering you.

**Comment by AlonVDev** [comment]
It is also the same problem for the Redstone particles.
